apps/pie-docs/src/_includes/nav.njk

```diff
@@ -16,6 +16,31 @@
 {% set navPagesTertiary = collections.all | eleventyNavigation("Docs-tertiary") %}
 {% set navSections = [navPagesPrimary, navPagesSecondary, navPagesTertiary] %}
 
+{% macro summary(title, iconKey) %}
+    {% set className = ('c-nav-category' if iconKey else 'c-nav-subCategory') %}
+    <summary class={{className}} style="--icon-fill: {{categoryIconFill}};">
+        {% if iconKey %}
+            {{ { name: navigationIcons[iconKey], attrs: { fill: categoryIconFill, height: '16', width: '16', class:'u-iconFilled', 'aria-hidden': 'true' } } | pieIconsSvg | safe }}
+        {% endif %}
+        {{ title }}
+        {{ { name: 'chevron-up', attrs: { fill: chevronFill, height: '16', width: '16', class:'c-nav-toggle c-nav-toggle--close', 'aria-hidden': 'true' } } | pieIconsSvg | safe }}
+        {{ { name: 'chevron-down', attrs: { fill: chevronFill, height: '16', width: '16', class:'c-nav-toggle c-nav-toggle--open', 'aria-hidden': 'true' } } | pieIconsSvg | safe }}
+    </summary>
+{% endmacro %}
+
+{% macro listItem(title, url, pageUrl, isActive) %}
+    <li>
+        <a
+            href="{{ url | url}}"
```

Comment:
isn't this doing the same thing 👀

Reply:
so the first url will be different from the second url?

I see it was `href="{{ subpage.url | url}}"` before